bootstrap/secret/secret.go

```diff
@@ -72,6 +72,18 @@ func NewSecretProvider(
 				secretClient, err = secrets.NewSecretsClient(ctx, secretConfig, lc, secureProvider.DefaultTokenExpiredCallback)
 				if err == nil {
 					secureProvider.SetClient(secretClient)
+
+					lc.Infof("SecretsFile is '%s'", secretConfig.SecretsFile)
```

Comment:
would suggest putting this in debug level instead of info as it is kind of sensitive

Reply:
Updated